gym/envs/classic_control/utils.py

```diff
@@ -0,0 +1,50 @@
+"""
+Utility functions used for classic control environments.
+"""
+
+from typing import Optional, Union
+
+import numpy as np
+
+
+def verify_number(x: Union[int, float, np.ndarray]) -> bool:
```

Comment:
really like your suggestion, thanks!
i've modified the util function so that it tries to do this casting and then returns the casted float. much cleaner, thanks!